packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -291,10 +303,7 @@ export class PHPRequestHandler implements RequestHandler {
 		if (this.php.fileExists(resolvedFsPath)) {
 			return resolvedFsPath;
 		}
-		if (!this.php.fileExists(`${this.#DOCROOT}/index.php`)) {
-			throw new Error(`File not found: ${resolvedFsPath}`);
-		}
-		return `${this.#DOCROOT}/index.php`;
+		throw new Error(`File not found: ${resolvedFsPath}`);
```

Comment:
This might be problematic. Let's still fall back to `index.php` in the rewrite handler.